packages/database/src/getConnectionConfig.js

```diff
@@ -3,27 +3,40 @@
  * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
  */
 
-// Note: Must use function to guarantee environment variables have loaded.
+const getServerConfig = () => ({
+  host: process.env.DB_URL,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_NAME,
+  ssl:
+    process.env.DB_ENABLE_SSL === 'true'
+      ? {
+          // Test server cannot turn on ssl, so sets the env to disable it
+          rejectUnauthorized: false,
+        }
+      : null,
+});
+
+const getCiConfig = () => ({
+  host: process.env.CI_TEST_DB_URL,
+  user: process.env.CI_TEST_DB_USER,
+  password: process.env.CI_TEST_DB_PASSWORD,
+  database: process.env.CI_TEST_DB_NAME,
+  ssl: null,
+});
+
+const validateEnv = () => {
+  ['DB_URL', 'DB_USER', 'DB_PASSWORD', 'DB_NAME'].forEach(requiredField => {
+    if (!process.env[requiredField]) {
+      throw new Error(
+        `Invalid database connection configuration: please specify '${requiredField}' in a .env file`,
+      );
+    }
+  });
+};
+
 export const getConnectionConfig = () => {
-  return process.env.CI_NAME === 'codeship'
-    ? {
-        host: process.env.CI_TEST_DB_URL,
-        user: process.env.CI_TEST_DB_USER,
-        password: process.env.CI_TEST_DB_PASSWORD,
-        database: process.env.CI_TEST_DB_NAME,
-        ssl: null,
-      }
-    : {
-        host: process.env.DB_URL,
-        user: process.env.DB_USER,
-        password: process.env.DB_PASSWORD,
-        database: process.env.DB_NAME,
-        ssl:
-          process.env.DB_DISABLE_SSL === 'true'
-            ? null
-            : {
-                // Test server cannot turn on ssl, so sets the env to disable it
-                rejectUnauthorized: false,
-              },
-      };
+  validateEnv();
+  // Note: Must use functions to guarantee environment variables have loaded
```

Comment:
Indeed this is the case... In previous commits, I removed the comment (and used `const`) without a problem, but I was probably lucky

This functionality now fails if functions are not used to retrieve the `.env` variables